### PR TITLE
[Feature][Github] Add stale mechanism to auto mark issues and PRs as stale

### PR DESCRIPTION
**Pull request by davidzollo**
Add stale mechanism to auto mark issues and PRs as stale

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          close-issue-message: >
+            This issue has been closed because it has not received response for too long time. You could
+            reopen it if you encountered similar problems in the future.
+          # Stale PRs
+          days-before-pr-stale: 120
+          days-before-pr-close: 7
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it has not had recent
+            activity for 120 days. It will be closed in 7 days if no further activity occurs.
+          close-pr-message: >
```
Comment (CalvinKirs) [inline .github/workflows/stale.yml:49]: We have a long list of updated tasks, and it's best to add this to guide new contributors to the next contribution

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          # Stale Issues
+          days-before-issue-stale: 30
+          days-before-issue-close: 7
+          # We do not stale Issues with label `Waiting for reply`, `new feature` and `STIP`
+          exempt-issue-labels: 'Waiting for reply,new feature,STIP,security'
```
Comment (Hisoka-X) [inline .github/workflows/stale.yml]: add `Waiting for code update` and `Waiting for users feedback` `feature`.

Comment (davidzollo) [reply .github/workflows/stale.yml]: has updated, PTAL

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**